canonical-websites/tutorials.ubuntu.com
# Add ./run command!

- Add ./run command, which runs through Docker
- Add a polymer wrapper script to set a port if it is in the env vars
- Update documentation
- Removed `build` script for now as it auto ran with `./run` and complicated it.


## QA
- `./run` should start up example content
- `./run serve [FILE or FOLDER]` should serve a specific location
- It should run on port 8088 as standard, and show in console output
